filebeat/docs/fields.asciidoc

```diff
@@ -1070,6 +1070,180 @@ Fields from AWS logs.
 
 
 
+[float]
+=== cloudtrail
+
+Fields for AWS CloudTrail logs.
+
+
+
+*`aws.cloudtrail.event_version`*::
++
+--
+The CloudTrail version of the log event format.
+
+
+type: keyword
+
+--
+
+[float]
+=== user_identity
+
+The userIdentity element contains details about the type of IAM identity that made the request, and which credentials were used. If temporary credentials were used, the element shows how the credentials were obtained.
+
+
+*`aws.cloudtrail.user_identity.type`*::
++
+--
+The type of the identity
+
+
+type: keyword
+
+--
+
+*`aws.cloudtrail.user_identity.arn`*::
++
+--
+The Amazon Resource Name (ARN) of the principal that made the call.
+
+type: keyword
+
+--
+
+*`aws.cloudtrail.user_identity.access_key_id`*::
++
+--
+The access key ID that was used to sign the request.
+
+type: keyword
+
+--
+
+[float]
+=== session_context
+
+If the request was made with temporary security credentials, an element that provides information about the session that was created for those credentials
+
+
+*`aws.cloudtrail.user_identity.session_context.mfa_authenticated`*::
++
+--
+The value is true if the root user or IAM user whose credentials were used for the request also was authenticated with an MFA device; otherwise, false.
+
+type: keyword
+
+--
+
+*`aws.cloudtrail.user_identity.session_context.creation_date`*::
++
+--
+The date and time when the temporary security credentials were issued.
+
+type: keyword
```

Comment:
Perhaps this should be an actual `date` type?

Reply:
yes it should